packages/react-core/src/components/DualListSelector/DualListSelector.tsx

```diff
@@ -299,13 +522,40 @@ export class DualListSelector extends React.Component<DualListSelectorProps, Dua
       addSelected,
       onListChange,
       id,
+      isTree,
+      hasChecks,
       ...props
     } = this.props;
-    const { availableOptions, chosenOptions, chosenOptionsSelected, availableOptionsSelected } = this.state;
+    const {
+      availableOptions,
+      chosenOptions,
+      chosenOptionsSelected,
+      availableOptionsSelected,
+      chosenTreeOptionsSelected,
+      availableTreeOptionsSelected
+    } = this.state;
+
     const availableOptionsStatusToDisplay =
-      availableOptionsStatus || `${availableOptionsSelected.length} of ${availableOptions.length} items selected`;
+      availableOptionsStatus ||
+      (isTree
```

Comment:
this is not i18n friendly (feel free to open followup issues as needed)

Reply:
The custom string `availableOptionStatus` should overwrite either of the default strings. What about it isn't i18n friendly?

Reply:
oh sorry, i didn't see that